yesod-persistent/yesod-persistent.cabal

```diff
@@ -13,12 +13,13 @@ homepage:        http://www.yesodweb.com/
 description:     API docs and the README are available at <http://www.stackage.org/package/yesod-persistent>
 extra-source-files: README.md ChangeLog.md
 
+flag persistent_has_persistent_template
+  default: True
```

Comment:
IIRC, this should have `manual: False`, right?

Reply:
Interesting ... the automatic flag selection doesn't seem to work in stack. At least that is how I interpret the CI failures. Does stack have a different default for manual than cabal-install does?

Reply:
Stack doesn't do dependency solving, and therefore it would need to be told to override any defaults set in the cabal file.